docs/taskruns.md

```diff
@@ -590,11 +593,26 @@ order to terminate the sidecars they will be restarted with a new
 Pod will include the sidecar container with a Retry Count of 1 and
 with a different container image than you might be expecting.
 
+<<<<<<< HEAD
+## Deleting completed TaskRun
+Deleting completed TaskRun automatically after *ExpirationSecondsTTL* time.
+
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: TaskRun
+metadata:
+  name: go-example-git
+spec:
+  # […]
+  expirationSecondsTTL: 300
+```
+=======
 Note: The configured "nop" image must not provide the command that the
 sidecar is expected to run. If it does provide the command then it will
 not exit. This will result in the sidecar running forever and the Task
 eventually timing out. https://github.com/tektoncd/pipeline/issues/1347
 is the issue where this bug is being tracked.
+>>>>>>> fa1704dac6afad20b5beee2c4bbc9ab2b0eb50ae
```

Comment:
just a quick ping - looks like there were some merge conflicts in this PR that didn't quite get resolved :pray:

Reply:
Thx, I'm going to check any other problems like this, and then update.